app/js/services/etablissementService.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+// Make sure to require Mingo for the browser
+var mingo = require('mingo/dist/mingo');
```

Comment:
Does this mean we could explicit files which requires `_` ?

Reply:
We can now use `require()` anywhere 🙂